srcpkgs/audacity/template

```diff
@@ -18,12 +19,14 @@ homepage="https://www.audacityteam.org"
 distfiles="https://github.com/${pkgname}/${pkgname}/archive/Audacity-${version}.tar.gz"
 checksum=759e8efd5e4ff6cc2fdd31b36bc631307330c1a6e2d3426e3f1d46ebcf8907a3
 
+export WX_CONFIG=wx-config-gtk3
```

Comment:
What also works is `ac_cv_path_WX_CONFIG=wx-config-gtk3` in `configure_args`. ;)